rocketchat-apb/roles/deprovision-rocketchat-apb/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+##############################################################################
+## Deprovision rocketchat-apb
+## This role executes much of the needed functionality to deprovision an
+## application using an Ansible Playbook Bundle and delete resources created
+## in the sample apb generated by the provision role.
+##############################################################################
+
+
+#############################################################################
+# Deprovision a route
+#############################################################################
+- openshift_v1_route:
+    name: rocketchat-apb
+    namespace: '{{ namespace }}'
+    state: absent
+
+
+#############################################################################
+# Deprovision a service
+#############################################################################
+- k8s_v1_service:
+    name: rocketchat-apb
+    namespace: '{{ namespace }}'
+    state: absent
+
+
+#############################################################################
```

Comment:
probably want to edit this comment now that we don't need the extra resource cleanup